test/kubemark/run-e2e-tests.sh

```diff
@@ -44,7 +44,7 @@ fi
 
 if [[ -f /.dockerenv ]]; then
 	# Running inside a dockerized runner.
-	go run ./hack/e2e.go -- -v --check-version-skew=false --test --test_args="--e2e-verify-service-account=false --dump-logs-on-failure=false ${ARGS}"
+	go run ./hack/e2e.go -- --check-version-skew=false --test --test_args="--e2e-verify-service-account=false --dump-logs-on-failure=false ${ARGS}"
```

Comment:
@shyamjvs I think this command need to live inside kubetest, I feel like the root issue is kubetest used in e2e.go is not the same with kubetest in the kubekins image